cross/libudev/Makefile

```diff
@@ -0,0 +1,97 @@
+PKG_NAME = libudev
+PKG_REAL_NAME = systemd
+PKG_VERS = 204
```

Comment:
Trying to remember why I selected release 204?
1. because this was the minimum version needed for fuse3?
2. because that is the default version for DSM7?
```
# udevadm --version
219
```
3. because that is the default version for DSM6?  `<--- probably it`
```
# udevadm --version
204
```

Question ends-up being, should we align `udev` with DSM releases?  I would believe it would be best?

Reply:
And version 204 is also on my DS210+ with DSM 5.2

Reply:
comcerto2k-7.0 builds now with libudev_219 too, but I don't know, wheter systemd 219 or 204 is installed.

Reply:
I've asked on discord... Might get something from there on our support channel...